### ISSUE TITLE
vsf not working with docker installation guide / tutorial

### ISSUE THREAD

**Issue by apaajabolehd** [DivanteLtd/vue-storefront#4522]
## Current behavior

I have successfully installed the vsf-api and vsf without any problem. But when I tried to run the http://localhost:3000, the server is not running because an error. When I checked the logs with "docker-compose logs -f" at "vue-storefront" folder, there is a missing theme required, "vsf-default" or "vsf-capybara". 

So I tried to install the vsf-default theme, and successfully running the and access the http://localhost:3000 (with "yarn dev", "docker-compose up -d" in tutorial not working). 

But it always redirect to http://localhost:3000/error everytime I refresh any pages. 
And this is the error log I found on vsf log :

`[search] Can not connect the vue-storefront-api / ElasticSearch instance! Error: {"code":500,"result":{"root_cause":[{"type":"index_not_found_exception","reason":"no such index","resource.type":"index_or_alias","resource.id":"vue_storefront_catalog","index_uuid":"_na_","index":"vue_storefront_catalog"}],"type":"index_not_found_exception","reason":"no such index","resource.type":"index_or_alias","resource.id":"vue_storefront_catalog","index_uuid":"_na_","index":"vue_storefront_catalog"}}` 

## Steps to reproduce the issue

just follow this tutorial "install with docker" part.
https://docs.vuestorefront.io/guide/cookbook/setup.html#_0-introduction

## Can you handle fixing this bug by yourself?

- [ ] YES
- [x] NO

## Which [Release Cycle](https://docs.vuestorefront.io/guide/basics/release-cycle.html) state this refers to? Info for developer.
Pick one option.

- [ ] This is a bug report for test version on https://test.storefrontcloud.io - In this case Developer should create branch from `develop` branch and create Pull Request `2. Feature / Improvement` back to `develop`.
- [ ] This is a bug report for current Release Candidate version on https://next.storefrontcloud.io - In this case Developer should create branch from `release` branch and create Pull Request `3. Stabilisation fix` back to `release`.
- [x] This is a bug report for current Stable version on https://demo.storefrontcloud.io and should be placed in next stable version hotfix - In this case Developer should create branch from `hotfix` or `master` branch and create Pull Request `4. Hotfix` back to `hotfix`.

## Environment details

- Browser: Google Chrome, Version 83.0.4103.61 (Official Build) (64-bit)
- OS: Mac OS Mojave, 10.14.5 
- Node: 10.15.3
- Code Version: vsf-api 29d4ce5, vsf d04b90f

**Comment by gibkigonzo** [comment]
Hey, you are trying to fetch data from ES with index `vue_storefront_catalog`. Check if you indeed have index `vue_storefront_catalog_${timestamp}` and alias `vue_storefront_catalog`. You can use https://chrome.google.com/webstore/detail/elasticsearch-head/ffmkiejjmecolpfloofpjologoblkegm to check you ES connection

**Comment by dvanmali** [comment]
I have run the same instructions from the tutorial through my docker instance. My error output looks like this if this helps with debugging

`$ cross-env TS_NODE_PROJECT="tsconfig-build.json" ts-node ./core/scripts/generate-files.ts`
`WARNING: NODE_ENV value of 'docker' did not match any deployment config file names.`
`WARNING: See https://github.com/lorenwest/node-config/wiki/Strict-Mode`
`The theme you want to use does not exist.`
`Please use 'vsf init' or install manually one of our official themes:`
`- https://github.com/DivanteLtd/vsf-capybara#--installation`
`- https://github.com/DivanteLtd/vsf-default#--installation`
`error Command failed with exit code 1.`
`info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.`

**Comment by apaajabolehd** [comment]
@gibkigonzo I've tried the ES add-on, and yeah, it seems thats the problem, my ES health status is not connected. So I tried run `yarn restore` and its running now. But why the documentation not mention it at all? and do I have run it everytime I up the container?

**Comment by apaajabolehd** [comment]
oh I forgot to ask this, I see in tutorial that I just need to run `docker-compose up -d`, but when I tried to run the front server with `docker-compose up -d` and then run `docker ps`, I see the container is running but the port is empty for `vue-storefront_app` image. I tried open localhost:3000, nothing run on that server.

Currently, I'm using `yarn dev` to run the front server.

**Comment by pkarw** [comment]
@kkdg can you take a look?

**Comment by pkarw** [comment]
@kkdg is working on docs update on docker - should be covered with this new deploy shortly

**Comment by py-lot** [comment]
I spend a week dealing with this issue and no idea how fix it... any info from the documentation.
For somebody like me with no idea about the project is hard to follow and get it up and running vue-storefront. Please update the https://docs.vuestorefront.io/guide/cookbook/setup.html#_0-introduction
thanks

**Comment by CombatCode** [comment]
Exactly the same issues on the clean installation of latest stable version. Seriously, dealing with this project second day in a row and nothing is working here correctly.  FFS who is using it

**Comment by afnineone** [comment]
> @gibkigonzo I've tried the ES add-on, and yeah, it seems thats the problem, my ES health status is not connected. So I tried run `yarn restore` and its running now. But why the documentation not mention it at all? and do I have run it everytime I up the container?

Could you further explain how exactly you fixed the ES problem? And in what directory did you run "yarn restore"?

**Comment by Cool-and-Deadly** [comment]
I'm having the same error as @dvanmali on a fresh installation running `docker-compose up -d`
Any help will be much appreciated
